docs/plugins/transform-response-cookie.md

```diff
@@ -0,0 +1,63 @@
+## Transform response cookie plugin
+
+`transform-response-cookie` plugin allows setting of any attributes of cookies encoded in Set-Cookie, which are sent out through Pyron.
+Changes, as specified in the plugin's conf, will be applied to the cookies matching the filter.
+Filter is based on "name", "domain" and "path" attributes.
+The name is mandatory, domain and path are optional.
+If not set, they will match any value.
+If explicitly set to null, they will match only cookies that don't contain this attribute.
+
+
+Enable `transform-response-cookie` plugin by adding `plugin/transform-response-cookie` to `MODULES` environment variable.
+
+#### Example configuration may look like so:
+
+```json
+{
+  "rules": [
+    {
+      "default": {
+        "targetHost": "example.com",
+        "targetPort": 80
+      },
+      "endpoints": [
+        {
+          "method": "GET",
+          "pathPattern": "/transform-some-cookies",
+          "responsePlugins": [
+            {
+              "name": "transform-response-cookie",
+              "conf": {
+                "name": "originalCookieName",
+                "domain": "optional.domain.com",
+                "path": "/optional/path/",
+                "body": {
```

Comment:
"body" should be not part of 'conf', compare with TransformResponseCookieConf case class